src/Microsoft.AspNetCore.Authentication/AuthenticationSchemeOptions.cs

```diff
@@ -36,5 +37,21 @@ public virtual void Validate(string scheme)
         /// If set, will be used as the service type to get the Events instance instead of the property.
         /// </summary>
         public Type EventsType { get; set; }
+
+        // Scheme forwarding properties
+
+        public string ForwardDefault { get; set; }
```

Comment:
These are going to need thorough explanations, they're very meta.

Does ForwardDefault make sense for anything but the virtual handler? If you set it on something like Google then none of Google's functions would work, right?